### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by craigs40**
This project is a restructuring of the Awesome Books app code. The goal is to make it more organized by using modules and ES6 best practices:
- Use ES6 modules to write modular JavaScript.
- Learn how to use proper ES6 syntax.
- Understand improvements that ES6 brings to JavaScript.
- Use npm as software packages system.

### CODE REVIEW

**Review by rotimiazeez — APPROVED** [review]
# Hi @craigs40 👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🎉🎉
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+title.addEventListener('change', () => {
+  inputBook.title = title.value;
+});
+
+author.addEventListener('change', () => {
+  inputBook.author = author.value;
+});
+
+Book.displayBooks();
+populateFields();
```
Comment (FrederickMih) [inline index.js:108]: - [x] Please let's further divide our codes into resealable modules(ie small units of independent, reusable code). Remember that the main objective of this project is to understand how to use ES6 Modules. For example, `class Book{}`, and the methods `removeBook{}`, `addBook{}`, `displayBooks{}` can be use as a modules.

**Review by FrederickMih — COMMENTED** [review]
#### Hi @craigs40 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mmsesay — COMMENTED** [review]
Hi @craigs40, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that have been reviewed, approved and merge. If you are repeating the week, please create a new and open PR and ask for a review when ready. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**